Swiftfin.xcodeproj/project.pbxproj

```diff
@@ -2023,6 +2043,40 @@
 			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-Swiftfin iOS/Pods-Swiftfin iOS-frameworks.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
+		AB001A5552FE000244A447A4 /* [CP] Prepare Artifacts */ = {
```

Comment:
1 - iOS isn't building because `VLCMediaPlayer.swift` was added to the extensions target as well, that must be removed.
2 - do a `pod install` so that these are removed as these shouldn't ever be added